src/online-stats.jl

```diff
@@ -47,14 +44,14 @@ function Series(x::Columns, y::AbstractVector, stats::OnlineStat{(1,0), 1}...)
 end
 
 """
-`aggregate_stats(series::AbstractSeries, t::Union{IndexedTable, DTable)`
+`aggregate_stats(series::AbstractSeries, t::Union{IndexedTable, DTable; by, with)`
```

Comment:
I also removed AbstractSeries.  There's only Series now.

Reply:
Cool, my packages were in a bad state. Updated again.